test/index.js

```diff
@@ -60,23 +60,38 @@ describe('Schwifty', () => {
         useNullAsDefault: true
     };
 
-    const getServer = (options, cb) => {
+    async function getServer (options) {
 
         const server = new Hapi.Server();
```

Comment:
Let's start writing it `Hapi.server()` since that is way chill.

Reply:
ahhh!!! very cool, I'd totally missed that. Thanks for that.